app/views/articles/show.html.erb

```diff
@@ -195,6 +195,7 @@
       <% cache("article-bottom-content-#{rand(5)}-#{@article.id}-#{user_signed_in?}-#{(@organization || @user).latest_article_updated_at}", expires_in: 15.minutes) do %>
         <% @display_ad = DisplayAd.for_display(area: "post_comments",
                                                user_signed_in: user_signed_in?,
+                                               user_id: current_user&.id,
```

Comment:
`current_user` in these views is currently not possible, because of edge caching, effectively meaning that this PR depends on #19227 before it can go further.